src/components/announcementsList/index.tsx

```diff
@@ -109,7 +110,13 @@ const AnnouncementsList: React.FC<AnnouncementsListProps> = ({
         <>
           <AnnouncementsListWrapper>
             {currentAnnouncements.map((announcement, i) => {
-              return <AnnouncementCard {...announcement} key={i} />;
+              return (
+                <AnnouncementCard
+                  {...announcement}
+                  imageSrc={announcement.imageSrc || DEFAULT_IMAGE}
```

Comment:
Same comment as below, if the card already checks if imageSrc is undefined and replaces it then we dont need to do it here.